appservice/multiplexer.py

```diff
@@ -218,5 +230,5 @@ def watch_redis():
     try:
         httpd.serve_forever()
     except KeyboardInterrupt:
-        proc.kill()
-        proc.wait()
+        NGINX_PID.kill()
+        NGINX_PID.wait()
```

Comment:
This can't be correct as `start_nginx` returns `proc.pid`, so this seems broken to me. We would need to rename `NGINX_PID` to `NGINX_PROC`.

Reply:
Oops, well spotted! Fixed.